geodistpy/distance.py

```
@@ -3,22 +3,30 @@
-This implementation provides a fast computation of geo-spatial distances in comparison to alternative methods for computing geodesic distance (tested: geopy and GeographicLib, see `geokernels.test_geodesics` for test functions).
+This implementation provides a fast computation of geo-spatial distances in comparison
+to alternative methods for computing geodesic distance
+(tested: geopy and GeographicLib, see `tests.test_geodist` for test functions).
 
 ## References:
 
 - [Vincenty's Formulae](https://en.wikipedia.org/wiki/Vincenty's_formulae)
 - [GeographicLib](https://geographiclib.sourceforge.io/)
-- Karney, Charles F. F. (January 2013). "Algorithms for geodesics". Journal of Geodesy. 87 (1): 43–55. [arXiv:1109.4448](https://arxiv.org/abs/1109.4448). Bibcode:2013JGeod..87...43K. [doi:10.1007/s00190-012-0578-z](https://doi.org/10.1007/s00190-012-0578-z). Addenda.
+- Karney, Charles F. F. (January 2013). "Algorithms for geodesics".
+  Journal of Geodesy. 87 (1): 43–55.
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)